lib/aws_loader.rb

```diff
@@ -1,9 +1,9 @@
 require 'yaml'
-require_relative '../config/s3_store'
 module AwsLoader
   
-  AWS_PATH = File.join(QIt.root, 'config/aws.yml')
+  AWS_PATH = File.join(Dir.pwd, 'config/aws.yml')
   AWS = YAML.load(File.read(AWS_PATH))["aws"] if File.exists?(AWS_PATH)
+
   def generate_aws_yml_file
     sample_aws = {"aws"=>{"access_key_id"=>"", "secret_access_key"=>"", "region"=>""}}
     File.open(AWS_PATH, 'w') do |file|
```

Comment:
When there is no`config` folder, this line fails and the program terminates. Check the existence of the path and create if required.